.github/workflows/build_wheels.yml

```diff
@@ -4,6 +4,9 @@ on:
   release:
     types:
       - published
+  pull_request:
```

Comment:
Now that everything works fine, you could remove these lines :)

Reply:
Just removed that commit